iac/membership/main.tf

```diff
@@ -0,0 +1,23 @@
+resource "github_membership" "member" {
```

Comment:
nit: I would put this under `tools/...`, e.g. `tools/org_settings`, I think `iac` is unnecessarily cryptic